docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -158,11 +158,17 @@ For more information, see the link:https://github.com/zchee/docker-machine-drive
 To use Minishift with Hyper-V:
 
 . Install link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
+
+. Add the user to the local Hyper-V Administrators group.
++
+NOTE: These permissions are needed to allow the user to create and delete virtual machines with the Hyper-V Management API. 
+For more information, see xref:../roubleshooting/troubleshooting-driver-plugins.adoc#hyper-v-commands-must-be-run-as-an-administrator[Hyper-V commands must be run as an Administrator]
```

Comment:
Typo as mentioned by @apupier later part from `#` should need to be change to point to exact location in the page.
```
xref:../troubleshooting/troubleshooting-driver-plugins.adoc#insufficient-privileges[Hyper-V commands must be run as an Administrator]
```